samples/nrf_desktop/src/hw_interface/motion_optical.c

```diff
@@ -631,6 +686,14 @@ static int init(u16_t cpi)
 		goto error;
 	}
 
+	/* Set initial power mode downshift times. */
+	update_downshift_time(OPTICAL_REG_RUN_DOWNSHIFT,
+			      CONFIG_DESKTOP_OPTICAL_RUN_DOWNSHIFT_TIME_MS);
```

Comment:
I think we should pass it as an argument from the caller (just like with cpi)
this way we set it to default value in one place only

Reply:
(also for other values)

Reply:
done